### PR TITLE
UICIRC-912: Use camel case notation for all data-testid

### PR DESCRIPTION
**Pull request by Dmitriy-Litvinenko**
## Purpose
- Use camel case notation for all data-testid
- Remove magic strings for testIds and labelIds

## Refs
https://issues.folio.org/browse/UICIRC-912

### CODE REVIEW

File: src/settings/LoanPolicy/components/ViewSections/RequestManagementSection/RequestManagementSection.test.js [hunk 1]
```diff
@@ -83,7 +88,7 @@ describe('RequestManagementSection', () => {
     });
 
     it('should render component header', () => {
-      expect(getById('sectionHeaderTestId').getByText(requestManagementLabelId)).toBeVisible();
+      expect(getById('sectionHeaderTestId').getByText(labelIds.requestManagementLabelId)).toBeVisible();
```
Comment (artem-blazhko) [inline src/settings/LoanPolicy/components/ViewSections/RequestManagementSection/RequestManagementSection.test.js:91]: Could you use constant?

File: src/settings/LoanPolicy/components/ViewSections/RequestManagementSection/RequestManagementSection.test.js [hunk 2]
```diff
@@ -119,7 +124,7 @@ describe('RequestManagementSection', () => {
...
```
Comment (artem-blazhko) [inline src/settings/LoanPolicy/components/ViewSections/RequestManagementSection/RequestManagementSection.test.js:127]: Could you use constant?

File: src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js [hunk 3]
```diff
@@ -49,9 +54,9 @@ describe('RadioGroup', () => {
   });
 
   it('should render label of component correctly', () => {
-    expect(getById('chargeAmountItemTestId').getByText(chargeAmountItemLabelId)).toBeVisible();
-    expect(screen.getByTestId('chargeAmountItemTestId')).toHaveAttribute('for', 'chargeAmount');
-    expect(screen.getByTestId('chargeAmountItemTestId')).toHaveAttribute('id', 'chargeAmount-label');
+    expect(getById('chargeAmountItemTestId').getByText(labelIds.chargeAmountItemLabelId)).toBeVisible();
```
Comment (artem-blazhko) [inline src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js:57]: Could you use constant here?
Also, I would suggest to do searching through the application and try to find `getById` custom method which uses for finding elements by `data-testid`.
Here you missed it and I think you could miss it in other places.

File: src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js [hunk 4]
```diff
@@ -63,7 +68,7 @@ describe('RadioGroup', () => {
       type: 'radio',
     };
 
-    expect(getById('actualCostTestId').getByText(actualCostLabelId)).toBeVisible();
+    expect(getById('actualCostTestId').getByText(labelIds.actualCostLabelId)).toBeVisible();
```
Comment (artem-blazhko) [inline src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js:71]: Could you use constant here?

File: src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js [hunk 5]
```diff
@@ -75,13 +80,13 @@ describe('RadioGroup', () => {
       type: 'radio',
     };
 
-    expect(getById('setCostTestId').getByText(setCostLabelId)).toBeVisible();
+    expect(getById('setCostTestId').getByText(labelIds.setCostLabelId)).toBeVisible();
```
Comment (artem-blazhko) [inline src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js:83]: Could you use constant here?

File: src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js (hunk 3 above)
Comment (artem-blazhko) [reply src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js:57]: I see 99 matches in 19 files.
image

**Review by s3fs — APPROVED** [review]
Could you please fix Artem's comments?

File: src/settings/LoanPolicy/components/ViewSections/RequestManagementSection/RequestManagementSection.test.js (hunk 1 above)
Comment (Dmitriy-Litvinenko) [reply src/settings/LoanPolicy/components/ViewSections/RequestManagementSection/RequestManagementSection.test.js]: Fixed

File: src/settings/LoanPolicy/components/ViewSections/RequestManagementSection/RequestManagementSection.test.js (hunk 2 above)
Comment (Dmitriy-Litvinenko) [reply src/settings/LoanPolicy/components/ViewSections/RequestManagementSection/RequestManagementSection.test.js]: Fixed

File: src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js (hunk 3 above)
Comment (Dmitriy-Litvinenko) [reply src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js]: Fixed

File: src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js (hunk 4 above)
Comment (Dmitriy-Litvinenko) [reply src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js]: Fixed

File: src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js (hunk 5 above)
Comment (Dmitriy-Litvinenko) [reply src/settings/LostItemFeePolicy/components/EditSections/RadioGroup/RadioGroup.test.js]: Fixed

**Review by artem-blazhko — APPROVED** [review]

**Pull request merged**